src/main/java/com/automationexercise/Pages/HomePage.java

```diff
@@ -0,0 +1,29 @@
+package com.automationexercise.Pages;
+
+import com.automationexercise.Utilities.Utility;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import static com.automationexercise.Utilities.Utility.clicking;
+import static com.automationexercise.Utilities.Utility.verifyElementVisible;
+
+
+public class HomePage {
+
+    private final WebDriver driver;
+    private final By testcasebtn_ele = By.xpath("(//a[@href=\"/test_cases\"])[1]");
```

Comment:
* Use correct naming conventions for **testcasebtn_ele** to be **testCaseButton**
* Make the XPath the last thing to use, start with id, name,class, CSS selector